sections/slideshow.liquid

```diff
@@ -0,0 +1,445 @@
+              sizes="100vw"
+              src="{{ block.settings.image | img_url: '1500x' }}"
+              loading="lazy"
+              alt="{{ block.settings.image.alt | escape }}"
+              width="{{ block.settings.image.width }}"
+              height="{{ block.settings.image.width | divided_by: block.settings.image.aspect_ratio | round }}"
+            >
+          {%- else -%}
+            {{ 'lifestyle-2' | placeholder_svg_tag: 'placeholder-svg' }}
+          {%- endif -%}
+        </div>
+        <div class="slideshow__text-wrapper slideshow__text-wrapper--{{ section.settings.box_align }} banner__content page-width">
+          <div class="slideshow__text banner__box color-{{ section.settings.color_scheme }}">
+            {%- if block.settings.heading != blank -%}
+              <h2 class="banner__heading {% if section.settings.heading_size == 'medium' %}h1{% else %}h0{% endif %}">{{ block.settings.heading | escape }}</h2>
+            {%- endif -%}
+            {%- if block.settings.subheading != blank -%}
+              <div class="banner__text" {{ block.shopify_attributes }}>
+                <span>{{ block.settings.subheading | escape }}</span>
+              </div>
+            {%- endif -%}
+            {%- if block.settings.button_label != blank -%}
+              <a{% if block.settings.link %} href="{{ block.settings.link }}"{% else %} role="link" aria-disabled="true"{% endif %} class="button {% if block.settings.button_style_secondary %}button--secondary{% else %}button--primary{% endif %}">{{ block.settings.button_label | escape }}</a>
+            {%- endif -%}
+          </div>
+        </div>
+      </div>
+    {%- endfor -%}
+  </div>
+
+  {%- if section.blocks.size > 1 -%}
+    <div class="slideshow__controls slider-buttons no-js-hidden color-{{ section.settings.color_scheme }}">
+      <button type="button" class="slider-button slider-button--prev" name="previous" aria-label="{{ 'accessibility.previous_slideshow' | t }}" aria-controls="Slider-{{ section.id }}">{% render 'icon-caret' %}</button>
+      <div class="slider-counter{% if section.settings.slider_visual == 'counter' or section.settings.slider_visual == 'numbers' %} caption{% endif %}">
+        {%- if section.settings.slider_visual == 'counter'-%}
+          <span class="slider-counter--current">1</span>
+          <span aria-hidden="true"> / </span>
+          <span class="visually-hidden">{{ 'accessibility.of' | t }}</span>
+          <span class="slider-counter--total">{{ section.blocks.size }}</span>
+        {%- else-%}
+          <div class="slideshow__control-wrapper" role="group" aria-label="{{ 'accessibility.load_slide' | t }}">
+            {%- for block in section.blocks -%}
+              <button class="slider-counter__link slider-counter__link--{{ section.settings.slider_visual }} link" aria-label="{{ forloop.index }} {{ 'accessibility.of' | t }} {{ forloop.length}}" aria-controls="Slider-{{ section.id }}">
```

Comment:
From Fable session, let's adjust the `aria-label` to be more clear and precise on what these controls do.

For example, an accName of `1 of 3` did not provide enough detail to convey the purpose of the control. Adding more detail, such as `Load slide 1 of 3` is more informative.

Reply:
Should I update the copy to say that a slide is active/loaded ? Say if I'm on slide 1 already, should the aria-label say `Loaded slide 1 of 3` or something along those lines ? Though I guess it should announce the new slide when shown into the view. It doesn't seem to do it atm though 🤔